dipy/align/streamwarp.py

```diff
@@ -278,8 +278,13 @@ def bundlewarp_shape_analysis(moving_aligned, deformed_bundle, no_disks=10,
 
     for i in range(n):
 
-        shape_profile[i] = np.mean(offsets[indx == i])
-        stdv[i] = np.std(offsets[indx == i])
+        mask = indx == i
+        if sum(mask):
+            shape_profile[i] = np.mean(offsets[mask])
+            stdv[i] = np.std(offsets[mask])
+        else:
+            shape_profile[i] = np.nan  # or pass and leave it to zero ?
```

Comment:
If the case makes sense, should these be left to their original zero values, and use a simple `pass` instead?

Reply:
@BramshQamar, can you answer @jhlegarreta  concerning this point.

Reply:
@jhlegarreta @skoudoro 
We should keep this line because 0 values imply minimum to no shape differences in the two bundles' segments. This would be the case when looking at shape differences between two identical bundles.

Reply:
The expected values of the analysis should be explained in the docstring: what a 0 means, what an `np.nan` means, and what any value in between means.

Kept `np.nan`  and removed the temporary comment in 48a6110.